### PR TITLE
Display submitted-to proposals (and document version) on document overview.

### PR DESCRIPTION
**Pull request by deiferni**
Also display a link to directly update the document in its proposal when it is
outdated.

screen shot 2015-04-16 at 17 01 19

### CODE REVIEW

File: opengever/document/locales/de/LC_MESSAGES/opengever.document.po [hunk 1]
```diff
@@ -88,6 +92,10 @@ msgstr "Diese Meldung nicht mehr anzeigen."
 msgid "Logout"
 msgstr "GEVER verlassen"
 
+#: ./opengever/document/browser/overview_templates/overview.pt:31
+msgid "Outdated, update document in proposal"
+msgstr "Veraltete eingereichte Version aktualisieren"
```
Comment (phgross) [inline opengever/document/locales/de/LC_MESSAGES/opengever.document.po]: `Veraltetet, eingereichte Version aktualisieren.`

File: opengever/document/locales/de/LC_MESSAGES/opengever.document.po [hunk 2]
```diff
@@ -103,6 +111,10 @@ msgstr "Als E-Mail versenden"
 msgid "Submit additional documents"
 msgstr "Dokumente nachreichen"
 
+#: ./opengever/document/browser/overview_templates/overview.pt:20
+msgid "Submitted in Proposals"
+msgstr "Eingereicht bei"
```
Comment (phgross) [inline opengever/document/locales/de/LC_MESSAGES/opengever.document.po]: The Default says no the same than the translation.

Comment (deiferni) [inline opengever/document/locales/de/LC_MESSAGES/opengever.document.po]: Submitted with

**Comment by deiferni** [comment]
@phgross updated

**Comment by phgross** [comment]
please rebase ...

**Comment by phgross** [comment]
:+1: 

Styling improvements will follow in a separate PR.

**Pull request merged**